apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -38,15 +38,17 @@ const (
 	SidecarVersionEnvVar             string = "TAG"
 	SidecarProviderArgsEnvVar        string = "PROVIDER_ARGS"
 	SidecarDefaultSyncProviderEnvVar string = "SYNC_PROVIDER"
+	SidecarLogFormatEnvVar           string = "LOG_FORMAT"
 	defaultSidecarEnvVarPrefix       string = "FLAGD"
 	InputConfigurationEnvVarPrefix   string = "SIDECAR"
 	defaultMetricPort                int32  = 8014
 	defaultPort                      int32  = 8013
 	defaultSocketPath                string = ""
 	defaultEvaluator                 string = "json"
 	defaultImage                     string = "ghcr.io/open-feature/flagd"
-	// `INPUT_FLAGD_VERSION` is replaced in the `update-flagd` Makefile target
-	defaultTag             string           = "INPUT_FLAGD_VERSION"
+	// `v0.3.4` is replaced in the `update-flagd` Makefile target
+	defaultTag             string           = "v0.3.4"
```

Comment:
```suggestion
	// `INPUT_FLAGD_VERSION` is replaced in the `update-flagd` Makefile target
	defaultTag             string           = "INPUT_FLAGD_VERSION"
```

This is automatically updated during the build process.

Reply:
Yes in fact it will fail the build (I believe we have some check around this).